dataland-frontend/src/components/pages/DataPrivacy.vue

```diff
@@ -1,38 +1,1534 @@
+          text-indent: -1cm;
+          line-height: 133%;
+        "
+      >
+        9.<span style="font: 7pt &quot;Times New Roman&quot;"
+          >&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span
+        >PROFILING
+      </h1>
+
+      <p class="MsoNormal" style="margin-bottom: 12pt; line-height: 133%">
+        <span style="font-size: 10pt; line-height: 133%"
+          >Wir verarbeiten Ihre personenbezogenen Daten nicht automatisiert in einer Weise, dass sie Ihnen gegenüber
+          rechtliche Wirkung entfaltet oder sie in ähnlicher Weise erheblich beeinträchtigt.</span
+        >
+      </p>
+
+      <h1
+        style="
+          margin-top: 0cm;
+          margin-right: 0cm;
+          margin-bottom: 12pt;
+          margin-left: 1cm;
+          text-align: justify;
+          text-indent: -1cm;
+          line-height: 133%;
+        "
+      >
+        10.<span style="font: 7pt &quot;Times New Roman&quot;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>AKTUALITÄT
+        UND ÄNDERUNG DIESER DATENSCHUTZERKLÄRUNG
+      </h1>
+
+      <p class="MsoNormal" style="margin-bottom: 12pt; line-height: 133%">
+        <span style="font-size: 10pt; line-height: 133%"
+          >Diese Datenschutzerklärung ist aktuell gültig und wurde das letzte Mal im Oktober 2023 aktualisiert.</span
+        >
+      </p>
+
+      <p class="MsoNormal" style="margin-bottom: 12pt; line-height: 133%">
+        <span style="font-size: 10pt; line-height: 133%">&nbsp;</span>
+      </p>
+
+      <p class="MsoNormal">&nbsp;</p>
     </div>
-  </div>
-  <TheFooter />
+  </LegalPage>
 </template>
 
 <script lang="ts">
-import BackButton from "@/components/general/BackButton.vue";
-import MarginWrapper from "@/components/wrapper/MarginWrapper.vue";
-import TheFooter from "@/components/generics/TheFooter.vue";
+import LegalPage from "@/components/generics/LegalPage.vue";
 
 export default {
   name: "DataPrivacy",
-  components: {
-    BackButton,
-    MarginWrapper,
-    TheFooter,
-  },
+  components: { LegalPage },
 };
 </script>
+
+<style scoped>
+/* Font Definitions */
+@font-face {
+  font-family: Gulim;
+  panose-1: 2 11 6 0 0 1 1 1 1 1;
+}
+
+@font-face {
+  font-family: "Cambria Math";
+  panose-1: 2 4 5 3 5 4 6 3 2 4;
+}
+
+@font-face {
+  font-family: "Segoe UI";
+  panose-1: 2 11 5 2 4 2 4 2 2 3;
+}
+
+@font-face {
+  font-family: "Noto Sans Symbols";
+}
+
+@font-face {
+  font-family: "Century Gothic";
+  panose-1: 2 11 5 2 2 2 2 2 2 4;
+}
+
+@font-face {
+  font-family: "Arial Fett";
+  panose-1: 0 0 0 0 0 0 0 0 0 0;
+}
+
+@font-face {
+  font-family: Georgia;
+  panose-1: 2 4 5 2 5 4 5 2 3 3;
+}
+
+@font-face {
+  font-family: "\@Gulim";
+  panose-1: 2 11 6 0 0 1 1 1 1 1;
+}
+
+/* Style Definitions */
+p.MsoNormal,
+li.MsoNormal,
+div.MsoNormal {
+  margin-top: 0cm;
+  margin-right: 0cm;
+  margin-bottom: 8pt;
+  margin-left: 0cm;
+  line-height: 107%;
+  font-size: 11pt;
+  font-family: "Arial", sans-serif;
+}
+
+h1 {
```

Comment:
It feels to me like we have a lot of repeating styles associated with each h1 block an d that these styles could just instead moved here saving a lot of duplication. Same for the other styles.

Reply:
Should we really do this? I'm asking because it is more of a readability change on generated code.

Reply:
If I'm imagining the AGBs being like a 10'000 LOC file, I overall feel like we shouldn't use the Word HTML conversion feature in the future anymore but rather have a proper html file and update the html directly in the future whenever the AGBs, Data Privacy etc changes.

I.e. I don't think we should treat this as regenerated code. I don't think we should generate this code again in the future.

Reply:
New formatting doesn't work on mobile:

![image](https://github.com/d-fine/Dataland/assets/35459207/916a0acb-0f29-42bc-9911-e1917610cdf8)